GoogleUtilities/Example/Tests/Swizzler/GULAppDelegateSwizzlerTest.m

```diff
@@ -60,7 +60,12 @@ @interface GULTestAppDelegate : UIResponder <UIApplicationDelegate> {
 /** A URL property that is set by the app delegate methods, which is then used to verify if the app
```

Comment:
I might not be seeing it due to weird github diffs, but if it doesn't already exist, can you add a test that makes sure `proxyOriginalDelegateIncludingAPNSMethods` doesn't get swizzled when swizzling is disabled?